static/js/src/modules/search-panel.js

```diff
@@ -0,0 +1,45 @@
+const searchPanel = () => {
+  /**
+    Sets up the search handler.
+  */
+  const form = document.querySelector('[data-js="search-form"]'),
+    input = document.querySelector('[data-js="form-text"]'),
+    header = document.querySelector('[data-js="navigation"]'),
+    searchClose = document.querySelector('[data-js="search-close"]');
+
+  input.addEventListener('focus', e => {
+    header.classList.add('search-active');
+  });
+
+  if (searchClose !== null) {
+    searchClose.addEventListener('click', e => {
+      e.preventDefault();
+      header.classList.remove('search-active');
+      input.value = '';
```

Comment:
If there is a previous query do we maybe want to keep that in the search box?

Reply:
Yep, good idea, I won't clear it.